pkg/networkservice/common/connect/client_map.go

```diff
@@ -36,7 +36,11 @@ func (c *clientMap) Delete(key string) {
 
 func (c *clientMap) Load(key string) (networkservice.NetworkServiceClient, bool) {
 	value, ok := c.internalMap.Load(key)
-	return value.(networkservice.NetworkServiceClient), ok
```

Comment:
https://github.com/golang/go/issues/29763

Reply:
Ah... in the event we get back a nil, we can't type coerce that... good catch.